README.md

```diff
@@ -147,61 +147,18 @@ https://github.com/angular/angular-cli/issues/6349).
 
 ### Updating the API using swagger-codegen
 
-We use [swagger-codegen](https://github.com/swagger-api/swagger-codegen) to automatically implement the API, as defined in `api/jobs.yaml`, for all
-servers and the UI. Whenever the API is updated, follow these steps to update the server implementations. 
-
-**Note:** after updating the API files you will need to test and update the server implementations to resolve any broken dependencies on old 
-API definitions or implement additional functionality to match the new specs. 
-
-From the base of the checked-out job-manager repository, run:
-```sh
-scripts/rebuild_swagger.sh
-```
-
-This will:
- - Find and download `swagger-codegen-cli.jar`
- - Remove the necessary build artifacts 
- - Use `swagger-codegen-cli.jar` to regenerate them. 
-
-If you prefer to do this manually you can fallow the instructions below.
-
-#### Manual Process
+We use [swagger-codegen](https://github.com/swagger-api/swagger-codegen) to automatically generate the API classes
+defined in `api/jobs.yaml` for the servers and for the UI. 
+Whenever the API is updated, re-run `docker-compose up rebuild-swagger` to trigger a rebuild.
+ 
+**Note:** the `swagger-rebuild` job does nothing if the file `api/jobs.yaml` has not changed since the
```

Comment:
isn't it `rebuild-swagger`  instead of `swagger-rebuild` ?

Reply:
Also – if `docker-compose up` will automatically run `rebuild-swagger` if needed, do we need this note?

Reply:
This note is still useful in case you want to rebuild the swagger model without also bringing up the service.

Say you just made a change to the model in a way that you know will break the UI and the services. You'll want to rebuild everything from `jobs.yaml` but there's no point bringing up the other services that you know will now have compilation errors.